server/circleci.go

```diff
@@ -76,51 +75,63 @@ func (s *Server) triggerCircleCiIfNeeded(pr *model.PullRequest) {
 	mlog.Info("Triggered circleci", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number), mlog.String("fullname", pr.FullName))
 }
 
+func (s *Server) requestEETriggering(ctx context.Context, pr *model.PullRequest, eeBranch string, triggerBranch string) error {
+	circleciClient := http.DefaultClient
```

Comment:
If we are passing the default client, then there's no need to pass it really. We can just use that directly in that method.